spaghetto/shell.nix

```diff
@@ -27,5 +27,6 @@ mkShell {
     easy-ps.purescript-language-server
     easy-ps.purs-tidy
     nodejs
+    esbuild
```

Comment:
I moved this to the shell instead of npm dependencies, since it's not really needed there.